apis/python/src/tiledbsoma/_tdb_handles.py

```diff
@@ -419,6 +409,64 @@ def enum(self, label: str) -> tiledb.Enumeration:
         raise NotImplementedError
 
 
+class DataFrameWrapper(SOMAArrayWrapper, Wrapper[clib.SOMADataFrame]):
+    """Wrapper around a Pybind11 SOMADataFrame handle."""
+
+    @classmethod
+    def _opener(
+        cls,
+        uri: str,
+        mode: options.OpenMode,
+        context: SOMATileDBContext,
+        timestamp: int,
+    ) -> clib.SOMADataFrame:
+        open_mode = clib.OpenMode.read if mode == "r" else clib.OpenMode.write
+        config = {k: str(v) for k, v in context.tiledb_config.items()}
+        column_names: List[str] = []
+        result_order = clib.ResultOrder.automatic
+        return clib.SOMADataFrame.open(
+            uri,
+            open_mode,
+            config,
+            column_names,
+            result_order,
+            (0, timestamp),
```

Comment:
once again going to bang on my drum about passing these as named arguments, since after `uri` and `open_mode`, there is no obvious order for any of them to be in. it looks like the `column_names` and `result_order` variables above may exist only to provide a name for passing these in, so that could be eliminated too:

```python
return ....open(
    uri,
    open_mode,
    config={k: str(v) for ...},
    column_names=[],
    result_order=clib.ResultOrder.automatic,
    timestamp=(0, timestamp),
)
```

Reply:
I thought I had changed this in Pybind11 to force these to be named arguments and error out if not passed in as kw args 🤔 I'll look again and at the very least make sure they're all named this time.